apps/docs/src/components/Hero/HeroWrapper.svelte

```diff
@@ -164,9 +164,7 @@
       in={2.7}
       out={3.9}
     >
-      <h2
-        class="fixed left-0 top-[66svh] top-[66vh] flex w-screen flex-col items-center justify-center"
-      >
+      <h2 class="fixed left-0 top-[66vh] flex w-screen flex-col items-center justify-center">
```

Comment:
The Tailwind extension is falsely complaining about it. If a browser doesn't know [dynamic viewport units](https://caniuse.com/viewport-unit-variants), it will fall back to regular units. Since dynamic viewport units have good support nowadays, we can probably reside to only use svh, but I wouldn't drop it, otherwise scrolling on an iPhone might result in jumping text (because of the appearing/disappearing status bar).

Reply:
good points! thank you